libs/fluid-elements/tab-group/src/lib/tab-group/tab-group.ts

```diff
@@ -46,6 +46,8 @@ import {
   FLUID_SPACING_0,
   FLUID_SPACING_MEDIUM,
 } from '@dynatrace/fluid-design-tokens';
+// TODO: Ask Lukaz or someone how to solve this
+import { getNextGroupItemIndex } from '@dynatrace/fluid-elements/core';
```

Comment:
@rowa-audil to solve what? :D

Reply:
@lukasholzer This is solved actually. I need to get back on looking over the PR's. 😠

Reply:
This is already solved 😄